src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,10 +2,24 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+
+    @ManyToMany
+    @JoinColumn(name = "movie_id")
```

Comment:
many-to-many must have `@JoinTable` instead, because a single column isn't enough to store all the needed relations